rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => !string.includes(char));
+
+const isSimpleString = string => doesNotContain(
+	string,
+	['^', '$', '+', '[', '{', '(', '\\', '.', '?', '*']
```

Comment:
@jfmengels I might do that at some point. I'm actually writing a blog post about it as we speak.